src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-policy/rgw-multisite-sync-policy.component.ts

```diff
@@ -0,0 +1,96 @@
+import { TitleCasePipe } from '@angular/common';
+import { Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { forkJoin } from 'rxjs';
+import { RgwMultisiteService } from '~/app/shared/api/rgw-multisite.service';
+import { CellTemplate } from '~/app/shared/enum/cell-template.enum';
+import { CdTableColumn } from '~/app/shared/models/cd-table-column';
+
+@Component({
+  selector: 'cd-rgw-multisite-sync-policy',
+  templateUrl: './rgw-multisite-sync-policy.component.html',
+  styleUrls: ['./rgw-multisite-sync-policy.component.scss']
+})
+export class RgwMultisiteSyncPolicyComponent implements OnInit {
+  @ViewChild('tooltipTmpl', { static: true })
+  public tooltipTmpl: TemplateRef<any>;
+
+  columns: Array<CdTableColumn> = [];
+  syncPolicyData: any = [];
+
+  constructor(
+    private rgwMultisiteService: RgwMultisiteService,
+    private titleCasePipe: TitleCasePipe
+  ) {}
+
+  ngOnInit(): void {
+    this.columns = [
+      {
+        name: $localize`Group Name`,
+        prop: 'groupName',
+        flexGrow: 1
+      },
+      {
+        name: $localize`Status`,
+        prop: 'status',
+        flexGrow: 1,
+        cellTransformation: CellTemplate.tooltip,
+        customTemplateConfig: {
+          map: {
+            Enabled: { class: 'badge-success', tooltip: 'sync is allowed and enabled' },
+            Allowed: { class: 'badge-info', tooltip: 'sync is allowed' },
+            Forbidden: {
+              class: 'badge-warning',
+              tooltip:
+                'sync (as defined by this group) is not allowed and can override other groups'
+            }
+          }
+        },
+        pipe: this.titleCasePipe
+      },
+      {
+        name: $localize`Zonegroup`,
+        prop: 'zonegroup',
+        flexGrow: 1
+      },
+      {
+        name: $localize`Bucket`,
+        prop: 'bucket',
+        flexGrow: 1
+      }
+    ];
+
+    forkJoin(
+      this.rgwMultisiteService.getSyncPolicy('', '', true),
+      this.rgwMultisiteService.getSyncPolicy('', '', false)
```

Comment:
what about adapting the API further to fetch everything at once, when the boolean is set? That would improve this code I guess.